tests/components/daikin/test_config_flow.py

```diff
@@ -82,21 +88,27 @@ async def test_discovery(hass, mock_daikin):
     flow = init_config_flow(hass)
 
     result = await flow.async_step_discovery({KEY_IP: HOST, KEY_MAC: MAC})
-    assert result["type"] == data_entry_flow.RESULT_TYPE_CREATE_ENTRY
+    assert result["type"] == RESULT_TYPE_CREATE_ENTRY
     assert result["title"] == HOST
     assert result["data"][CONF_HOST] == HOST
     assert result["data"][KEY_MAC] == MAC
 
 
 @pytest.mark.parametrize(
     "s_effect,reason",
-    [(asyncio.TimeoutError, "device_timeout"), (Exception, "device_fail")],
+    [
+        (asyncio.TimeoutError, "device_timeout"),
+        (HTTPForbidden, "forbidden"),
+        (ClientError, "device_fail"),
+        (Exception, "device_fail"),
+    ],
 )
 async def test_device_abort(hass, mock_daikin, s_effect, reason):
     """Test device abort."""
     flow = init_config_flow(hass)
     mock_daikin.factory.side_effect = s_effect
 
     result = await flow.async_step_user({CONF_HOST: HOST})
-    assert result["type"] == data_entry_flow.RESULT_TYPE_ABORT
-    assert result["reason"] == reason
+    assert result["type"] == RESULT_TYPE_FORM
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5518609)